### PR TITLE
Create: Add tables to DB

### PR DESCRIPTION
**Pull request by williamrolando88**
In this PR I:
- Created the structure for the tables on this database
- Create the connections based on the following schema: schema

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,71 @@
...
+ALTER TABLE invoice_items
+  ADD CONSTRAINT invoices_fk
+  FOREIGN KEY (invoice_id)
+  REFERENCES invoices(id);
+
+ALTER TABLE
+clinic_db=# ALTER TABLE invoice_items
+  ADD CONSTRAINT treatments_fk
+  FOREIGN KEY (treatment_id)
+  REFERENCES treatments(id);
```
Comment (MahmoudBakr23) [inline schema.sql:71]: - Kindly make sure that all the foreign keys have indexes. For example:
`
CREATE INDEX ON
medical_histories
(patient_id) ;
`

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,71 @@
...
+  treatment_id INT NOT NULL
 );
+
+CREATE TABLE treatments (
+  id SERIAL NOT NULL PRIMARY KEY,
+  type VARCHAR(50) NOT NULL,
+  name VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE medical_treatment (
```
Comment (MahmoudBakr23) [inline schema.sql:40]: - Please keep in mind that you still need to join tables that have many to many relationships (even if they're not shown in the diagram). For example:
`
CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
`

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @williamrolando88 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪

- Professional README file 
- Well done implementing the database schema following the diagram 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (williamrolando88) [reply schema.sql]: It was already implemented under the name of: medical_treatment

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @williamrolando88 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**